packages/renderer/src/stitch-frames-to-video.ts

```diff
@@ -359,7 +359,7 @@ const innerStitchFramesToVideo = async (
 			x264Preset,
 			colorSpace,
 		}),
-		codec === 'h264' ? ['-movflags', 'faststart'] : null,
+		codec === 'h264' || codec === 'h264-ts' ? ['-movflags', 'faststart'] : null,
```

Comment:
Again, I'm not sure if there are more possible codec values we need to handle here